i18n/zh/docusaurus-plugin-content-docs/current/userguide/failover/cluster-failover.md

```diff
@@ -136,6 +136,15 @@ JSONPath 表达式遵循 Kubernetes 规范：<https://kubernetes.io/docs/referen
 
 **注意**：JSONPath 表达式默认从 API 资源对象的 "status" 字段开始查找。例如，若要提取 Deployment 的 "availableReplicas"，应使用 `{.availableReplicas}`，而不是 `{.status.availableReplicas}`。
 
+#### 配置驱逐速率限制
+
+在多集群大规模故障的场景下，同时驱逐大量工作负载可能会对 Karmada 控制面以及剩余的健康集群造成巨大压力。为了提升故障迁移过程的稳定性和可控性，Karmada 引入了速率限制机制，可以根据需要通过命令行参数调整驱逐速率。
+
+用户可以通过在 `karmada-controller-manager` 组件上配置以下命令行参数，来精细化控制驱逐行为：
+
+* `--resource-eviction-rate`:
+  系统每秒驱逐速率。默认值为 **0.5**。
```

Comment:
The explanation of the parameters could be more detailed; providing an example would be even better.